operating/collector-runtime-configuration.adoc

```diff
@@ -0,0 +1,32 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="collector-runtime-configuration"]
+= Collector runtime configuration
+
+:FeatureName: Collector runtime configuration
+include::snippets/technology-preview.adoc[]
+:!FeatureName:
+
+toc::[]
+
+[role="_abstract"]
+Collector runtime configuration enables users to modify some collector behaviors without restarting Collector. Collector runtime configuration is set using a `ConfigMap` object called `collector-config`. Whenever you create or update the `ConfigMap` object, Collector refreshes the runtime configuration. When you delete the `ConfigMap` object, the settings revert to the default runtime configuration values. 
+
+Currently only two settings are controlled via Collector runtime configuration. `networking.externalIps.enabled` controls if the visualizing external entities feature is enabled or disabled. Please note that `networking.externalIps.enable` has been changed to `networking.externalIps.enabled`, and it has been changed from a boolean to an enum that has valid values of ENABLED and DISABLED. The default is DISABLED. Learn more about the visualizing external entities feature xref:../operating/visualizing-external-entities.adoc#visualizing-external-entities[here]. `networking.maxConnectionsPerMinute` is the maximum number of open networking connections reported by Collector per container per minute. The default value is 2048.
+
+The following is an example.
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: collector-config
+  namespace: stackrox
+data:
+  runtime_config.yaml: | <1>
+    networking:
+      externalIps:
+        enabled: ENABLED
+      maxConnectionsPerMinute: 2048
+----
+<1> {product-title-short} mounts this file at `/etc/stackrox/runtime_config.yaml`.
```

Comment:
` {product-title-short}` actually appears of the web page instead of being replaced by RHACS. I don't know why that is.

Reply:
I have fixed this.